tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts

```diff
@@ -148,6 +151,29 @@ describe('Graph Executions Container', () => {
       By.css('debug-tensor-value')
     );
     expect(debugTensorValueElements.length).toBe(tensorContainers.length);
+
+    // Clicking the tensor name buttons should dispatch the proper action.
```

Comment:
sounds like a name of a new behavior test spec.

Reply:
Done making this a standalone `it()`.